### PR TITLE
New rule: no-describe-variables

### PR DESCRIPTION
**Pull request by emilos**
Hey,

I've added a new linter rule that could help with memory leaks - please have a look and let me know what you think.

Best,
Emil

### CODE REVIEW

File: docs/rules/no-describe-variables.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+  it('test', function() {
+    expect(a).toBeDefined();
+  });
+});
+```
+
+or
+
+## Rule details
```
Comment (burabure) [inline docs/rules/no-describe-variables.md]: these examples are not clear to me, how do they relate to the former ones?

File: docs/rules/no-describe-variables.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+The following are considered warnings:
```
Comment (burabure) [inline docs/rules/no-describe-variables.md]: "The following are considered warnings/The following patterns are not warnings" structure should be used for all exmaples

**Comment by burabure** [comment]
specs errored out here https://travis-ci.org/tlvince/eslint-plugin-jasmine/jobs/245394318

File: docs/rules/no-describe-variables.md (hunk 1 above)
Comment (emilos) [inline docs/rules/no-describe-variables.md]: @burabure sorry, I've somehow missed one commit, added better examples

**Comment by emilos** [comment]
updated eslint to fix the travis issue, the tests are passing - I've installed the packages via npm5 instead of yarn and got local false positives :) is there anything else I should fix?

**Comment by burabure** [comment]
Thanks for your PR! Merging this now :tada:

**Pull request merged**